py3status/core.py

```diff
@@ -408,7 +409,12 @@ def notify_user(self, msg, level='error', rate_limit=None, module_name=''):
                 cmd = ['notify-send', '-u', DBUS_LEVELS.get(level, 'normal'),
                        '-t', '10000', 'py3status', msg]
             else:
-                cmd = ['i3-nagbar', '-m', msg, '-t', level]
+                config = self.i3status_thread.config
+                nagbar_font = config.get('py3status').get('nagbar_font')
```

Comment:
`nagbar_font = config.get('py3status').get('nagbar_font')`

will fail if `config['py3status']` is `None`. `nagbar_font = config.get('py3status', {}).get('nagbar_font')` would be fine but my other comment fixes this issue